tests/helpers.py

```diff
@@ -56,41 +54,62 @@ def create_resource(
     assert created_resource, (
         f"Failed to create resource {resource_name}"
     )
-    if wait:
-        assert ocs_obj.ocp.wait_for_resource(
-            condition=desired_status, resource_name=resource_name
-        ), f"{ocs_obj.kind} {resource_name} failed to reach"
-        f"status {desired_status}"
     return ocs_obj
 
 
-def create_pod(desired_status=constants.STATUS_RUNNING, wait=True, **kwargs):
+def wait_for_resource_state(resource, state):
+    """
+    Wait for a resource to get to a given status
+
+    Args:
+        resource (OCS obj): The resource object
+        state (str): The status to wait for
+
+    Returns:
+        bool: True if resource reached the desired state, False otherwise
+    """
+    try:
+        resource.ocp.wait_for_resource(
+            condition=state, resource_name=resource.name
+        )
+    except TimeoutExpiredError:
+        return False
```

Comment:
Add warning logger describing the timeout

Reply:
Done